pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -328,6 +328,7 @@ public boolean commitSegment(String rawTableName, final String committingSegment
     oldSegMetadata.setEndOffset(nextOffset);
     oldSegMetadata.setStatus(CommonConstants.Segment.Realtime.Status.DONE);
     oldSegMetadata.setEndTime(now);
+    // TODO Set download URL? Should these segments be downloaded via vip?
```

Comment:
This is a TODO because I realized that we are not setting the download URI for the new realtime data segments. I will add a ticket.